android/src/main/java/com/thebluealliance/androidclient/helpers/ThreadSafeFormatters.java

```diff
@@ -55,4 +55,8 @@ public static synchronized String formatDoubleOnePlace(double input) {
     public static synchronized String formatDoubleTwoPlaces(double input) {
         return DOUBLE_TWO_PLACES_FORMAT.format(input);
     }
+
+    public static synchronized String formatStat(double stat) {
```

Comment:
Fair enough. I guess my thinking is that we don't have to care what `formatStat` does as long as it's consistent and we use it for all stats. I'll make the change.